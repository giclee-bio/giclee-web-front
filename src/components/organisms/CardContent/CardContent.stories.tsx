import { Meta, StoryObj } from '@storybook/react'

import CardContent from './CardContent'

const meta: Meta<typeof CardContent> = {}

export default meta
type Story = StoryObj<typeof CardContent>

export const Index: Story = {
  args: {},
}
