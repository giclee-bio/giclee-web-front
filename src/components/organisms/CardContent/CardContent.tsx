const CardContent = () => {
  return <>CardContent</>
}

export default CardContent
