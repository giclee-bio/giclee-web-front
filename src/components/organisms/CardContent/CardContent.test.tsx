import { render } from '@testing-library/react'

import CardContent from './CardContent'

describe('CardContent', () => {
  it('snapshot', async () => {
    const { asFragment } = render(<CardContent />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
