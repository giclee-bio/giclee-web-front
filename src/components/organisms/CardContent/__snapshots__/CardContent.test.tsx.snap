// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContent snapshot 1`] = `
<DocumentFragment>
  <div
    class="h-[667px] w-[375px] rounded-lg bg-stone-800"
  >
    <div
      class="mt-16 flex flex-col items-center"
    >
      <img
        alt="nacal"
        class="rounded-full"
        data-nimg="1"
        decoding="async"
        height="72"
        loading="lazy"
        src="/_next/image?url=%2Fnacal.png&w=256&q=75"
        srcset="/_next/image?url=%2Fnacal.png&w=96&q=75 1x, /_next/image?url=%2Fnacal.png&w=256&q=75 2x"
        style="color: transparent;"
        width="72"
      />
      <h2
        class="mt-2 text-lg font-bold"
      >
        nacal
      </h2>
      <p
        class="mt-2 text-sm"
      >
        よろしくお願いします！
      </p>
    </div>
    <li
      class="mt-8 flex list-none items-center justify-center"
    >
      <img
        alt="nacal"
        class="rounded"
        data-nimg="1"
        decoding="async"
        height="36"
        loading="lazy"
        src="/_next/image?url=%2Fnacal.png&w=96&q=75"
        srcset="/_next/image?url=%2Fnacal.png&w=48&q=75 1x, /_next/image?url=%2Fnacal.png&w=96&q=75 2x"
        style="color: transparent;"
        width="36"
      />
      <p
        class="ml-4"
      >
        aaaa
      </p>
    </li>
    <li
      class="mt-8 flex list-none items-center justify-center"
    >
      <img
        alt="nacal2"
        class="rounded"
        data-nimg="1"
        decoding="async"
        height="36"
        loading="lazy"
        src="/_next/image?url=%2Fnacal.png&w=96&q=75"
        srcset="/_next/image?url=%2Fnacal.png&w=48&q=75 1x, /_next/image?url=%2Fnacal.png&w=96&q=75 2x"
        style="color: transparent;"
        width="36"
      />
      <p
        class="ml-4"
      >
        bbbb
      </p>
    </li>
    <li
      class="mt-8 flex list-none items-center justify-center"
    >
      <img
        alt="nacal3"
        class="rounded"
        data-nimg="1"
        decoding="async"
        height="36"
        loading="lazy"
        src="/_next/image?url=%2Fnacal.png&w=96&q=75"
        srcset="/_next/image?url=%2Fnacal.png&w=48&q=75 1x, /_next/image?url=%2Fnacal.png&w=96&q=75 2x"
        style="color: transparent;"
        width="36"
      />
      <p
        class="ml-4"
      >
        cccc
      </p>
    </li>
  </div>
</DocumentFragment>
`;
